### PR TITLE
Added hall of fame.

### PR DESCRIPTION
**Pull request by jojolepro**
hello there.

From https://community.amethyst-engine.org/t/hall-of-fame/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,6 +156,14 @@ Other places you might want to check out are [r/rust_gamedev][rg] and the [#rust
 
 **Note:** Any interaction with the Amethyst project is subject to our [Code of Conduct](https://github.com/amethyst/amethyst/blob/master/CODE_OF_CONDUCT.md).
 
+[![](https://sourcerer.io/fame/jojolepro/amethyst/amethyst/images/0)](https://sourcerer.io/fame/jojolepro/amethyst/amethyst/links/0)
```
Comment (LucioFranco) [inline README.md]: is there a reason its under your username?

Comment (jojolepro) [reply README.md]: Because it needed an account to use for the api calls, so i used mine. Is this an issue?

Comment (LucioFranco) [reply README.md]: No I dont think so just wanted to make sure that we aren't putting something on the readme that ties to only one person.

**Comment by Moxinilian** [comment]
General Kenobi, what is this?

**Comment by Mathspy** [comment]
There's also the [all-contributors](https://github.com/all-contributors/all-contributors/) standard that might be a little less "competitive" oriented and a lot more rewarding for people who don't specifically do "code"

**Comment by Moxinilian** [comment]
> 
> 
> There's also the [all-contributors](https://github.com/all-contributors/all-contributors/) standard that might be a little less "competitive" oriented and a lot more rewarding for people who don't specifically do "code"

This is much better to me. I dislike how competitive Sourcerer makes it look like. It really isn't what contributing is about.

**Comment by jojolepro** [comment]
Makes sense, but I don't see how we can fit 158 contributors on the readme, and keep that updated (seems to be manual updates https://allcontributors.org/)

**Comment by Moxinilian** [comment]
I don't see fit any contributor on the README personally.

**Comment by LucioFranco** [comment]
@jojolepro @Moxinilian maybe it doesn't make sense to add this then, what we could do is make a website page dedicated to contributors?

**Comment by Mathspy** [comment]
@jojolepro There's actually [a bot](https://allcontributors.org/docs/en/bot/overview) or a [CLI tool](https://allcontributors.org/docs/en/cli/overview) to automate the process. But yeah admittedly the way it's represented in the README works better for smaller scale projects and I think a dedicated webpage with a link at the bottom from README will be much more fitting (This is also allowed according to the specs)

>Use the project README (or another prominent public documentation page in the project) to recognize the contributions of members of the project community.

**Pull request closed without merging**